resources.go

```diff
@@ -146,6 +146,7 @@ func Provider() tfbridge.ProviderInfo {
 			"cloudflare_zone_lockdown":          {Tok: makeResource(mainMod, "ZoneLockdown")},
 			"cloudflare_zone_settings_override": {Tok: makeResource(mainMod, "ZoneSettingsOverride")},
 			"cloudflare_zone":                   {Tok: makeResource(mainMod, "Zone")},
+			"cloudflare_custom_ssl":             {Tok: makeResource(mainMod, "CustomSsl")},
```

Comment:
Is this a bugfix?

Reply:
yes - it was missing from the SDK generation so I added it now

Reply:
Okay. Might add that to the release notes as well.

Reply:
+1 will add when releasing to stop another set of builds